compose.yaml

```diff
@@ -16,11 +16,6 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data/
       - .data/db_dumps:/db_dumps
-    healthcheck:
-      test: pg_isready -U postgres -d {{ project_name }}
-      interval: 5s
-      timeout: 10s
-      retries: 120
```

Comment:
what's the reasoning behind this change?

Reply:
ok, I've seen instances where, for example, somebody bumps up the Postgres version in Docker and then somebody else can't figure out why their app won't start. They are rare though and usually require intervention either way, so I'm ok with this.